Sources/MapboxDirections/RouteOptions.swift

```diff
@@ -159,6 +163,13 @@ open class RouteOptions: DirectionsOptions {
      */
     open var includesExitRoundaboutManeuver = false
     
+    /**
+     A Boolean value indicating whether `Directions` can refresh time-dependent properties of the `RouteLeg`s of the resulting `Route`s.
+     
+     To refresh the `RouteLeg.expectedSegmentTravelTimes`, `RouteLeg.segmentSpeeds`, and `RouteLeg.segmentCongestionLevels` properties, use the `Directions.refreshRoute(responseIdentifier:routeIndex:fromLegAtIndex:completionHandler:)` method. This property is ignored unless `profileIdentifier` is `DirectionsProfileIdentifier.automobileAvoidingTraffic`.
+     */
+    open var refreshingEnabled = true
```

Comment:
The default values of these properties should match Directions API default values as much as possible. `refreshingEnabled` may not be relevant to applications that use the MapboxDirections library directly without the navigation SDK. So this property’s default value should be `false`, and NavigationRouteOptions in the navigation SDK would set the value to `true`.